src/_grid.scss

```diff
@@ -0,0 +1,67 @@
+/* @TODO move to variables/settings file */
+$divisions: 'one' 1, 'two' 2 , 'three' 3, 'four' 4, 'five' 5, 'six' 6, 'seven' 7, 'eight' 8, 'nine' 9, 'ten' 10, 'eleven' 11, 'twelve' 12;
+
+/* 12 column grid */
+.grid {
+    display: grid;
+    grid-template-columns: repeat(12, [col] 1fr);
+    grid-auto-rows: minmax(50px, auto);
+    grid-gap: 1em;
+}
+
+/* Dynamic grid */
+.autofit {
+    grid-template-columns: repeat(auto-fit, [col] minmax(300px, 1fr));
+}
+
+/* This will be redundant if we create a .flex utility class */
+.column {
+    display: flex;
+    justify-content: center;
+    align-items: center; 
+}
+
+/* @TODO Move to mixin file
+ * Mixin to create all of the column divisions
+ */
+@mixin column-divisions($prefix, $columns) {
```

Comment:
what is `prefix` for?

Reply:
The prefix just adds the prefix before the class name. So for example, for a desktop it adds 'desktop-' to the front of the class name.

That way you can specify specific columns and offsets for different device widths.